Cargo.toml

```diff
@@ -23,10 +23,109 @@ default-members = ["bin/hera"]
 # https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html
 resolver = "2"
 
+[profile.release]
+strip = "debuginfo"
+lto = "thin"
+panic = "unwind"
+codegen-units = 1
+incremental = false
+
+[workspace.dependencies]
+# Workspace
+op-net = { path = "crates/net" }
+kona-providers = { path = "crates/kona-providers" }
+rollup = { path = "crates/rollup" }
+ser = { path = "crates/ser" }
+
+# Optimism
+superchain = { version = "0.5", default-features = false }
+kona-primitives = { git = "https://github.com/anton-rs/kona", branch = "main", default-features = true }
+kona-derive = { git = "https://github.com/anton-rs/kona", branch = "main", default-features = true }
+
+# Alloy
+alloy = { version = "0.3.6", features = [
+    "contract",
+    "providers",
+    "provider-http",
+    "provider-ws",
+    "provider-ipc",
+    "signers",
+    "consensus",
+    "rpc-types",
+    "rpc-types-engine",
+    "network",
+    "ssz"
+] }
+alloy-primitives = { version = "0.8", features = ["serde"] }
+alloy-rlp = "0.3"
+op-alloy-protocol = { version = "0.2.12", default-features = false }
+op-alloy-rpc-types = { version = "0.2.12", default-features = false }
+op-alloy-rpc-types-engine = { version = "0.2.12", default-features = false }
+op-alloy-genesis = { version = "0.2.12", default-features = false }
+
+# Tokio
+tokio = { version = "1.21", default-features = false }
+
+# Serialization
+serde_json = "1"
+ethereum_ssz = "0.7.1"
+
+# Reth
+reth = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-chainspec = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-discv5 = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-execution-errors = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-execution-types = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-exex = { git = "https://github.com/paradigmxyz/reth", features = ["serde"], rev = "000b412" }
+reth-network-peers = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-node-api = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-node-ethereum = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-primitives = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-provider = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-revm = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-evm = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
+reth-tracing = { git = "https://github.com/paradigmxyz/reth", rev = "000b412" }
```

Comment:
Not a huge fan of using revisions here, but there isn't a tagged version with the most revent alloy versions we need.

Reply:
I'm sure they'll make a new one soon, it's been almost a month!